.github/workflows/java_nightly.yml

```diff
@@ -164,17 +134,18 @@ jobs:
               fi
             fi
             echo "Artifacts $FILE installed locally to ~/.m2/repository/org/apache/arrow"
+            tree ~/.m2/repository/org/apache/arrow
           done
           popd
-          echo "**************** 3 - Local repository to sync ****************"
           rm -rf repo
-          mkdir repo
-          cp -r ~/.m2/repository/org/apache/arrow repo
-          tree repo
+          mkdir -p repo/org/apache
+          if [ -d "$HOME/.m2/repository/org/apache/arrow" ]; then
+            cp -r ~/.m2/repository/org/apache/arrow repo/org/apache
+          fi
           echo "Repo configured with Arrow Java artifacts ready to sync"
```

Comment:
I assume you tested locally that this creates a working maven repo?

Reply:
Yes, `mvn install:install-file` create all the directory structure needed locally to sync